code/modules/medical/cloner_defects.dm

```diff
@@ -479,6 +543,18 @@ ABSTRACT_TYPE(/datum/cloner_defect/organ_damage)
 		. = ..()
 		src.owner.vdisfigured = TRUE
 
+/// Basically makes you bald. Truly the worst of the defects.
+/datum/cloner_defect/baldness
```

Comment:
Hard no. Erasing a large part of someone's character design at random is not a good or interesting defect.